cmd/client/main.go

```diff
@@ -86,6 +87,22 @@ func main() {
 	}
 }
 
+func GetMessage(line string) *protocol.Message {
+	parts := strings.SplitN(line, " ", 2)
+	command := parts[0]
+
+	if strings.HasPrefix(command, "/") {
+		switch command {
+		case "/kick":
+			return protocol.NewMessage(protocol.KickUser, parts[1])
+		}
+	} else {
+		return protocol.NewMessage(protocol.SendMessage, line)
+	}
```

Comment:
Why are you checking if have some prefix? The default case can be the `return protocol.NewMessage(protocol.SendMessage, line)`

```go
func GetMessage(line string) *protocol.Message {
	parts := strings.Split(line, " ")
	command := parts[0]

	switch command {
	case "/list":
		return protocol.NewMessage(protocol.ListUsers, "")
	default:
		return protocol.NewMessage(protocol.SendMessage, line)
	}
}
```